spec/nostra_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../app/nostra.rb'
+
+RSpec.describe Nostra do
+  context 'when importing 1money csv file' do
+    # NOTE: samples based on 1Money csv export data
+
+    # before do
+    #   account.add_transactions_by_csv('spec/fixture/1Money-sample.html')
+    # end
+
+    before do
+      # NOTE: this is where persistence will really come in to play
+      # for now, just modeling to prove out import functions
+      Nostra.import_csv('spec/fixture/1Money-sample.html')
```

Comment:
yes.  ROM can handle persistence of many kinds of data objects.  you could potentially have a repository of CSV data, etc.  It might even make sense to persist the imported CSV raw (or with minor processing), and then have Relation/Repository for getting the data out.  IDK, probably makes more sense to just parse out the CSV and persist as DB objects though... but you /could/ :)

Reply:
Yeah, I thought about just going raw too. Might be fun to play with some noSQL option or mongoDB as well.